### DISCUSSION TITLE
Can I open files with programs that are added to Bottles?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#149] [Q&A] [answered]
There are some files that I can only open with windows programs.
I want to right click on them and do "open with..." in a file manager and then open the file with Bottles

**Comment by mirkobrombin** [comment]
This feature is not available. Can you please open a Feature request?

**Comment by mirkobrombin** [comment, marked as answer]
Ref: https://github.com/bottlesdevs/Bottles/issues/132
